src/patternfly/components/Menu/menu.scss

```diff
@@ -617,11 +617,12 @@
 
 // TODO: standardize icon fitting
 // - Menu item action
+// TODO in breaking change - remove button styling here that is taken care of by favorite button no
 .#{$menu}__item-toggle-icon,
 .#{$menu}__item-action {
   &.pf-m-favorited,
   &.pf-m-favorited:hover,
-  &.pf-m-favorited .#{$button} {
+  &.pf-m-favorited .#{$button}:not(.pf-m-favorited) {
```

Comment:
This is fine but we could see if react wants to remove `.pf-m-favorited` from `menu__item-action` since it will be on the button now - then we wouldn't need to modify this rule.